src/PostgREST/App.hs

```diff
@@ -235,6 +235,14 @@ app dbStructure conf apiRequest =
                                           status = rangeStatus frm to (toInteger <$> tableTotal)
                                       in (status, contentRange)
 
+splitKeyValue :: BS.ByteString -> (BS.ByteString, BS.ByteString)
+splitKeyValue kv = (k, BS.tail v)
+  where (k, v) = BS.break (== '=') kv
```

Comment:
Initially I worried about `=` signs in the value itself but it's fine, since `break` splits around the first one only. Good to know about this function, I hadn't noticed it in base.